src/app/containers/SocialEmbed/common/styles.js

```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled';
+import { GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+
+/**
+ * MAX_WIDTH        Ensures all embeds assume the same width. (Tweets max-out
+ *                  at 500px, which is why this is set to 31.25rem.)
+ * LAZYLOAD_OFFSET  The distance in pixels above or below the viewport before
+ *                  an embed is allowed to load.
+ */
+const MAX_WIDTH = '31.25rem';
+export const LAZYLOAD_OFFSET = 250;
+
+const getWrapperHeightStyles = oEmbed => {
+  /**
+   * Adjust MIN_HEIGHT to configure the default minimum height of Social Embed
+   * wrappers. This helps reduce layout shift. It is not applied to fallbacks.
+   */
+  const MIN_HEIGHT = '18.75rem';
+  if (oEmbed?.height) return `min-height: ${oEmbed.height / 16}rem`;
```

Comment:
🤯  Has optional chaining been added to the Babel config?  This is my first time seeing it in simorgh

Reply:
Yes! It's included in `@babel/preset-env`. 😊